crates/bevy_app/src/event.rs

```diff
@@ -56,11 +56,23 @@ enum State {
 }
 
 /// An event collection that represents the events that occurred within the last two
-/// [Events::update] calls. Events can be cheaply read using an [EventReader]. This collection is
-/// meant to be paired with a system that calls [Events::update] exactly once per update/frame.
-/// [Events::update_system] is a system that does this. [EventReader]s are expected to read events
-/// from this collection at least once per update/frame. If events are not handled within one
-/// frame/update, they will be dropped.
+/// [`Events::update`] calls.
+/// Events can be written to in parallel using an [`EventWriter`]
```

Comment:
EventWriter cannot write events in parallel. It internally uses a `ResMut<Events<T>>`

Reply:
Resolved myself so we can get this merged asap.